### PR TITLE
Create simple zipcode object and add rough data file.

### PR DESCRIPTION
**Pull request by k-sharpe**

### CODE REVIEW

File: postal-injection/src/main/java/com/google/sps/servlets/zipcode.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package com.google.sps;
+
+import java.util.ArrayList;
+class zipcode {
```
Comment (andreahu12) [inline postal-injection/src/main/java/com/google/sps/servlets/zipcode.java:4]: classnames should be capitalized like `ZipCode`

Comment (andreahu12) [reply postal-injection/src/main/java/com/google/sps/servlets/zipcode.java:4]: also might want to make it public like `public class ZipCode`

File: postal-injection/src/main/java/com/google/sps/servlets/Order.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  public String getOrderID() {
+    return OrderID;
+  }
+
+  public String getZipcode() {
+    return zipcode;
+  }
+
+  public String warehouseLat() {
```
Comment (jennygaoo) [inline postal-injection/src/main/java/com/google/sps/servlets/Order.java:31]: change warehouseLat() to getWarehouseLat()

File: postal-injection/src/main/java/com/google/sps/servlets/Order.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+  public String getWarehouseLon() {
+    return warehouseLon;
+  }
+
+  public String getLat() {
+    return lat;
+  }
+
+  public String lon() {
```
Comment (jennygaoo) [inline postal-injection/src/main/java/com/google/sps/servlets/Order.java:43]: change lon() to getLon()

File: postal-injection/src/main/java/com/google/sps/servlets/Order.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
+package com.google.sps;
+
+public class Order {
```
Comment (andreahu12) [inline postal-injection/src/main/java/com/google/sps/servlets/Order.java:3]: Add javadoc explaining what Order is

File: postal-injection/src/main/java/com/google/sps/servlets/Order.java [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+  private String OrderID;
+  private String zipcode;
+  private String warehouseLat;
+  private String warehouseLon;
+  private String lat;
+  private String lon;
```
Comment (andreahu12) [inline postal-injection/src/main/java/com/google/sps/servlets/Order.java:10]: Spell out latitude and longitude. Verbose, but clearer.

File: postal-injection/src/main/java/com/google/sps/servlets/Order.java [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  private int weight;
+
+  public Order(String OrderID, String zipcode, String warehouseLat, String warehouseLon, String lat, String lon, int weight) {
+    this.OrderID = OrderID;
```
Comment (andreahu12) [inline postal-injection/src/main/java/com/google/sps/servlets/Order.java:14]: ditto: spell out latitude and longitude. applies to getter methods too.

File: postal-injection/src/main/java/com/google/sps/servlets/ShipEngine.java [hunk 7]
```diff
@@ -0,0 +1,23 @@
+package com.google.sps;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+
+public class ShipEngine {
```
Comment (andreahu12) [inline postal-injection/src/main/java/com/google/sps/servlets/ShipEngine.java:7]: Add javadoc explaining what ShipEngine does

File: postal-injection/src/main/java/com/google/sps/servlets/ShipEngine.java [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+  private static boolean initialized = false;
+  private static Collection<ZipCode> zipcodes;
+  public ShipEngine() {
```
Comment (andreahu12) [inline postal-injection/src/main/java/com/google/sps/servlets/ShipEngine.java:10]: nit: add newline between 
`private static Collection<ZipCode> zipcodes;`
`public ShipEngine() {`

File: postal-injection/src/main/java/com/google/sps/servlets/ShipEngine.java [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+    if (this.initialized)) {
+      throw new Exception("A ShipEngine has already been initialized.");
```
Comment (andreahu12) [inline postal-injection/src/main/java/com/google/sps/servlets/ShipEngine.java:12]: Use more specific IllegalStateException instead
https://docs.oracle.com/javase/7/docs/api/java/lang/IllegalStateException.html

File: postal-injection/src/main/java/com/google/sps/servlets/ZipCode.java [hunk 10]
```diff
@@ -0,0 +1,44 @@
+package com.google.sps;
+
+import java.util.ArrayList;
+public class ZipCode {
```
Comment (andreahu12) [inline postal-injection/src/main/java/com/google/sps/servlets/ZipCode.java:4]: Add javadoc for ZipCode class

**Comment by k-sharpe** [comment]
Closing this branch to adjust changes via the newer branch

**Pull request closed without merging**